wolox-books/src/App.vue

```diff
@@ -5,4 +5,72 @@
 
 <style lang="scss">
 @import 'scss/aplication.scss';
+
+.container {
```

Comment:
Separate all this classes in the scss files that match it as a component, for example, the buttons in a `/scss/components/_buttons.scss`